src/main/service/FirebaseConfig.ts

```diff
@@ -0,0 +1,15 @@
+import * as admin from 'firebase-admin';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+export function initFirebaseAdmin() {
+  admin.initializeApp({
+    serviceAccountId: process.env.FIREBASE_SERVICE_ACCOUNT_ID,
+    credential: admin.credential.cert({
+      projectId: process.env.FIREBASE_PROJECT_ID,
+      privateKey: process.env.FIREBASE_PRIVATE_KEY,
+      clientEmail: process.env.FIREBASE_CLIENT_EMAIL,
+    }),
+  });
+  return admin;
+}
```

Comment:
```suggestion
export function initFirebaseAdmin(): Auth {
  admin.initializeApp({
    serviceAccountId: process.env.FIREBASE_SERVICE_ACCOUNT_ID,
    credential: admin.credential.cert({
      projectId: process.env.FIREBASE_PROJECT_ID,
      privateKey: process.env.FIREBASE_PRIVATE_KEY,
      clientEmail: process.env.FIREBASE_CLIENT_EMAIL,
    }),
  });
  return admin.auth();
}
```

How about this? That way everything has types now and we don't need `any`

Reply:
Then also for `FirebaseAuth` it would look something like this
![image](https://user-images.githubusercontent.com/49849754/108643946-ac855100-7461-11eb-812e-7ebf33e5c80c.png)

Reply:
Good refactoring! thanks!